src/styles/box/stars.scss

```diff
@@ -0,0 +1,49 @@
+.product-card {
+  &__stars {
+    box-sizing: border-box;
+    margin: 0 4px 0 3px;
+    padding: 0;
+  }
+
+  &__star {
+    display: inline-block;
+    background-image: $star-bg;
+    background-size: contain;
+    background-repeat: no-repeat;
+    background-position: center;
+
+    width: $star-width;
+    height: $star-height;
+    cursor: pointer;
+  }
+
+  &:nth-child(1) {
+    &.star--active ~ .star {
+      background-image: $star-active-bg;
+    }
+  }
+
+  &:nth-child(2) {
+    &.star--active ~ .star {
+      background-image: $star-active-bg;
+    }
+  }
+
+  &:nth-child(3) {
+    &.star--active ~ .star {
+      background-image: $star-active-bg;
+    }
+  }
+
+  &:nth-child(4) {
+    &.star--active ~ .star {
+      background-image: $star-active-bg;
+    }
+  }
+
+  &:nth-child(5) {
+    &.star--active ~ .star {
+      background-image: $star-active-bg;
+    }
+  }
```

Comment:
The use of `&:nth-child(n)` selectors seems incorrect for styling stars, as it is unlikely that the stars are direct children of the `.product-card` element. Consider revising the selector logic to correctly target the star elements.